boris-client/main/boris.hs

```diff
@@ -264,7 +266,7 @@ renderBuildData r i =
      , mconcat ["end-at: ", maybe "n/a" renderTime . buildDataEndTime $ r]
      , mconcat ["heartbeat-at: ", maybe "n/a" renderTime . buildDataHeartbeatTime $ r]
      , mconcat ["duration: ", maybe "n/a" (uncurry renderDuration) $ (,) <$> buildDataStartTime r <*> buildDataEndTime r]
-     , mconcat ["log: ", maybe "n/a" (const $ "boris log " <> renderBuildId i) . buildDataLog $ r]
+     -- , mconcat ["log: ", maybe "n/a" (const $ "boris log " <> renderBuildId i) . buildDataLog $ r]
```

Comment:
I think dropping it is actually fine. It was important before because calls to cloudwatch would error out if the log was not there, where as new api can handle that because it knows about build.

Reply:
Ack will do. Should something replace this in `main` though? Perhaps I have paged too much of this out over the week but I'm figuring something in `main` should collect the log and provide it to the view?

Reply:
It doesn’t need it, there are really two options remove it all together, or always print “boris log <build id>”, it doesn’t need to be conditional on any data and doesn’t need anything else in main, it is just a pointer that you can run the log command to get the contents of the log.

Reply:
boris log build-id 


HTML formatting got me.

Reply:
Ok cool - that's prompted me to go fix `Log _i -> pure ()` :)